instrumentation/tomcat/tomcat-10.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/tomcat/v10_0/Tomcat10ResponseMutator.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.tomcat.v10_0;
+
+import io.opentelemetry.javaagent.bootstrap.http.HttpServerResponseMutator;
+import org.apache.coyote.Response;
+
+public class Tomcat10ResponseMutator implements HttpServerResponseMutator<Response> {
+  public static final Tomcat10ResponseMutator INSTANCE = new Tomcat10ResponseMutator();
```

Comment:
Nit: we usually use enum singletons for cases like this one
```suggestion
public enum Tomcat10ResponseMutator implements HttpServerResponseMutator<Response> {
  INSTANCE;
```

Reply:
Done.